<testsuite name="login.User is able to login to OrangeHRM application using correct credentials" tests="3" errors="1" failures="1" skipped="0" time="14.629518" timestamp="2022-03-15T22:31:26.539173" hostname="Reanolds-MacBook-Air.local"><testcase classname="login.User is able to login to OrangeHRM application using correct credentials" name="Navigate to the application -- @1.1 " status="passed" time="3.036081"><system-out>
<![CDATA[
@scenario.begin
  Scenario Outline: Navigate to the application -- @1.1 
    Given I navigate to "https://opensource-demo.orangehrmlive.com/index.php" ... passed in 3.036s
    Then I see proper "OrangeHRM" ... passed in 0.000s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:
Either redirection was performed, or condition is not correct
https://opensource-demo.orangehrmlive.com/index.php

]]>
</system-out></testcase><testcase classname="login.User is able to login to OrangeHRM application using correct credentials" name="Navigate to the application -- @1.2 " status="failed" time="3.002516"><failure type="AssertionError" message="">
<![CDATA[
Failing step: Then I see proper "Hello, World" ... failed in 0.000s
Location: features/test_scenarios/login.feature:6
Traceback (most recent call last):
  File "/Users/reanoldpiatrenka/PycharmProjects/OpenHRM/venv/lib/python3.9/site-packages/behave/model.py", line 1329, in run
    match.run(runner.context)
  File "/Users/reanoldpiatrenka/PycharmProjects/OpenHRM/venv/lib/python3.9/site-packages/behave/matchers.py", line 98, in run
    self.func(context, *args, **kwargs)
  File "features/steps/login_steps.py", line 22, in verify_page_title
    assert page_title == LoginPageLocators.login_page_title
AssertionError

Captured stdout:
Either redirection was performed, or condition is not correct
https://opensource-demo.orangehrmlive.com/index.php]]>
</failure><system-out>
<![CDATA[
@scenario.begin
  Scenario Outline: Navigate to the application -- @1.2 
    Given I navigate to "https://opensource-demo.orangehrmlive.com/index.php" ... passed in 3.002s
    Then I see proper "Hello, World" ... failed in 0.000s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:
Either redirection was performed, or condition is not correct
https://opensource-demo.orangehrmlive.com/index.php

]]>
</system-out></testcase><testcase classname="login.User is able to login to OrangeHRM application using correct credentials" name="Login to application -- @1.1 " status="failed" time="8.590921"><error type="TypeError" message="find_element() takes from 1 to 3 positional arguments but 33 were given">
<![CDATA[
Failing step: Then I see dashboard ... failed in 0.000s
Location: features/test_scenarios/login.feature:17
Traceback (most recent call last):
  File "/Users/reanoldpiatrenka/PycharmProjects/OpenHRM/venv/lib/python3.9/site-packages/behave/model.py", line 1329, in run
    match.run(runner.context)
  File "/Users/reanoldpiatrenka/PycharmProjects/OpenHRM/venv/lib/python3.9/site-packages/behave/matchers.py", line 98, in run
    self.func(context, *args, **kwargs)
  File "features/steps/login_steps.py", line 40, in verify_dashboard_visible
    dashboard_page.is_visible(DashboardPageLocators.dashboard_page_tab)
  File "/Users/reanoldpiatrenka/PycharmProjects/OpenHRM/features/pages/base_page.py", line 27, in is_visible
    element = WebDriverWait(self.driver, 10).until(EC.visibility_of_element_located(by_locator))
  File "/Users/reanoldpiatrenka/PycharmProjects/OpenHRM/venv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py", line 78, in until
    value = method(self._driver)
  File "/Users/reanoldpiatrenka/PycharmProjects/OpenHRM/venv/lib/python3.9/site-packages/selenium/webdriver/support/expected_conditions.py", line 125, in _predicate
    return _element_if_visible(driver.find_element(*locator))
TypeError: find_element() takes from 1 to 3 positional arguments but 33 were given

Captured stdout:
Either redirection was performed, or condition is not correct
https://opensource-demo.orangehrmlive.com/index.php]]>
</error><system-out>
<![CDATA[
@scenario.begin
  Scenario Outline: Login to application -- @1.1 
    Given I navigate to "https://opensource-demo.orangehrmlive.com/index.php" ... passed in 2.994s
    When I enter login "Admin" ... passed in 0.070s
    And I enter password "admin123" ... passed in 0.043s
    And I click on login button ... passed in 5.483s
    Then I see dashboard ... failed in 0.000s

@scenario.end
--------------------------------------------------------------------------------

Captured stdout:
Either redirection was performed, or condition is not correct
https://opensource-demo.orangehrmlive.com/index.php

]]>
</system-out></testcase></testsuite>